### PR TITLE
MULE-17797: FTP Write throws MULE:UNKNOWN error instead of FTP:ILLEGAL_PATH

### PR DESCRIPTION
**Pull request by gabrieldalborgo**

### CODE REVIEW

File: src/test/munit/ftp-write-test-case.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        </java:invoke-static>
+    </munit:after-suite>
+
+    <munit:test name="ftp-write-on-invalid-working-directory" expectedErrorType="FTP:ILLEGAL_PATH">
+        <munit:behavior>
+            <set-variable variableName="fileName" value="filePath.txt"/>
+        </munit:behavior>
+        <munit:execution>
+            <ftp:write config-ref="config-invalid-folder" path='#[vars.fileName]' createParentDirectories="false">
+                <ftp:content>
```
Comment (SebaElizalde) [inline src/test/munit/ftp-write-test-case.xml]: Can be shortened to <ftp:content>Any String</ftp:content>

File: src/test/munit/ftp-write-test-case.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    #[%dw 2.0
+                    output application/java
+                    ---
+                    "asd"]
+                </ftp:content>
+            </ftp:write>
+        </munit:execution>
+    </munit:test>
+
+    <munit:test name="fto-write-on-invalid-absolute-path-equals-to-working-directory" expectedErrorType="FTP:ILLEGAL_PATH">
```
Comment (SebaElizalde) [inline src/test/munit/ftp-write-test-case.xml]: I am not too sure about the name of this test, what do you mean by "equals-to-working-directory" at the end? I would leave it at "fto-write-on-invalid-absolute-path". Plus, typo in "ftO" :P

File: src/test/munit/ftp-write-test-case.xml (hunk 1 above)
Comment (gabrieldalborgo) [reply src/test/munit/ftp-write-test-case.xml]: Fixed

File: src/test/munit/ftp-write-test-case.xml (hunk 2 above)
Comment (gabrieldalborgo) [reply src/test/munit/ftp-write-test-case.xml]: Fixed

**Pull request merged**